CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## unreleased
```

Comment:
Actually as we never released 0.5.0, this can be released as 0.5.0, no need for a new version,